.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: barbershop
+
+on:
+  pull_request:
+    branches:
+      - main
+      - development
```

Comment:
add one more option: `release/`  - we will create 1-2 releases for you to understand a release strategy.